pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -40,6 +43,19 @@ type datasourceInfo struct {
 	SecretKey string
 }
 
+// cloudWatchExecutor executes CloudWatch requests.
+type cloudWatchExecutor struct {
+	*models.DataSource
+
+	ec2Client           ec2iface.EC2API
+	rgtaClient          resourcegroupstaggingapiiface.ResourceGroupsTaggingAPIAPI
+	logsClientsByRegion map[string]cloudwatchlogsiface.CloudWatchLogsAPI
+	mtx                 sync.Mutex
+
+	queuesByRegion map[string](chan bool)
+	queueLock      sync.Mutex
+}
+
```

Comment:
Why move this struct? If you don't have to move it, it makes the diff larger than it has to be.

Reply:
Wanted to have all the struct definitions at the top of the file

Reply:
@kaydelaney Please don't move things around if not necessary, since it makes it more difficult to diff and see what has changed.

Reply:
@kaydelaney As I said before, please move this and `newExecutor` back to their original locations, since moving them around makes it difficult to review the real changes. If you want to move them, please discuss it in another PR.